SU-HSDO/suhumsci
# SHS-4898: [EX] Add link to user guide on all content types

# READY FOR REVIEW

## Summary
Adds user guide group and user guide link to details menu in the node edit/add forms

## Steps to Test
1. Verify that the `User Guide` field group is open and the `User Guide` link opens a new tab/window to `https://hsweb.slite.page/p/5Qd9ZB~jGu/H-and-S-Drupal-Platform-User-Guide` for all content types in the following scenarios
2. Creating new content
3. Editing existing content
4. Using different themes, i.e. colorful, traditional

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
